### ISSUE TITLE
Amazon Blocked

### ISSUE THREAD

**Issue by nickp85** [WaLLy3K/wally3k.github.io#184]
Amazon is getting blocked from loading, seems to be coming from the phishing filter. Started a couple weeks ago. If you keep refreshing and trying PC vs mobile it does eventually load but clicking around triggers more errors. I suspect Amazon is using cloudfront to do something which is being blocked by this filter.

<img width="884" alt="image" src="https://github.com/WaLLy3K/wally3k.github.io/assets/59027847/94a23e20-ab4b-4ddc-a101-00320ae56ff6">

Match found in https://v.firebog.net/hosts/RPiList-Phishing.txt:
  d3ag4hukkh62yn.cloudfront.net

**Comment by trizt** [comment]
Seems Amazon Sweden is blocked too domain name amazon.se and subdomain www.amazon.se.

**Comment by iTim314** [comment]
Confirming that www.amazon.com is blocked via CNAME d3ag4hukkh62yn.cloudfront.net.

**Comment by Denny-89** [comment]
In the original RPiList list it was already fixed some days ago, now we just need to wait until the v.firebog.net mirrors the newest version...

**Comment by WaLLy3K** [comment]
There was a hiccup with my parser script that had one command return an error if run by `cron`, which I've tracked down and resolved — apologies to @RPiList who has had to field several issues regarding this (and other similar) false positive(s) after the [initial report](https://github.com/RPiList/specials/issues/1199). The parser/mirror will now resume its schedule of updating every two days.

Thankfully, I can confirm `d3ag4hukkh62yn.cloudfront.net` has been removed from https://v.firebog.net/hosts/RPiList-Phishing.txt with the `07AUG23` update.
